docs/user_guide/selection/DropHighPSIFeatures.rst

```diff
@@ -68,6 +68,21 @@ to the PSI value. The most commonly used thresholds are:
 - Below 10%, the variable has not experienced a significant shift.
 - Above 25%, the variable has experienced a major shift.
 - Between those two values, the shift is intermediate.
+- 'auto' threshold will be calculated based on size of dataset and number of bins.
+
+For 'auto' formula we are using chi2 approximation, proposed by B. Yurdakul:
+
+.. math::
+
+    threshold = \chi^2_{(q,B-1)} . (\frac{1}{N} + \frac{1}{M})
+
+where p - percentile (p-value), B - number of bins, N - size of basis dataset, N - size of test dataset.
```

Comment:
There is no p in the formula. Did you mean q?

Also, could we replace by a full sentence with verbs? something like this:

where q is the percentile, B is the number of bins, N is the... and M is ...